### ISSUE TITLE
Cannot reproduce the raw GenImage

### ISSUE THREAD

**Issue by anonymoussss** [gendetection/UnbiasedGenImage#2]
First of all, thank you for your work.
I am also using timm's res50 to reproduce the genImage, but except for stable 1.4, I can't reproduce the others. For example, using midjourney's training set for training, the evaluation on midjourney's test set is only 80+ acc. 
I would like to ask if you use the same training script for all subsets when training raw genImage? Are all training data resized to 256? or resized to different sizes?

**Comment by gendetection** [comment]
Hi!
For reproducing the results on the raw GenImage dataset, you can either follow their instructions here:
https://github.com/GenImage-Dataset/GenImage
or use our repo, which is almost the same code. You can find the corresponding command for all experiments in the folder of each detector:
https://github.com/gendetection/UnbiasedGenImage/tree/master/ResNet50

In the case of reproducing raw GenImage results, the command is:

`sh ./distributed_train.sh 2 classic <path_to_genimage_csv> <train_subset> \
--class-map <../../class_map.txt> -b 128 --model resnet50 --sched cosine --epochs 200 --lr 0.05 --amp --remode pixel \
--reprob 0.6 --aug-splits 3 --aa rand-m9-mstd0.5-inc1 --resplit --jsd --dist-bn reduce --num-classes 2 \
--output <result_folder> --experiment <exp_name> \
--base_path <prepended_to_paths_in_csv> \
--resume <checkpoint.pth.tar>`

**Comment by anonymoussss** [comment]
Thanks! I just found that the dataset I downloaded had an error. I will close this issue right now.
